docs/pages/2.usage.md

```diff
@@ -84,3 +90,56 @@ const user = useSupabaseUser()
 ```
 
 > Learn how to protect your routes by writing your own [auth middleware composable](/advanced#auth-middleware).
+
+## Server routes services
+
+> This section assumes you're familiar with [Nitro](https://v3.nuxtjs.org/guide/concepts/server-engine#standalone-server), the server engine powered by Nuxt.
+
+In order to provide utilities similar to `vue` composables (see first section) in [server routes](https://v3.nuxtjs.org/guide/features/server-routes/), this module exposes services providing the same functionnalities.
+
+### `serverSupabaseClient`
+
+This function is working similary as the [useSupabaseClient](/usage#usesupabaseclient) composable but can be used in server routes.
+
+Define your server route and just use the `serverSupabaseClient` like:
+
+```ts [server/api/librairies.ts]
+export default eventHandler(async (event) => {
+  const client = serverSupabaseClient(event)
+
+  const { data } = client.from('librairies').select('name')
+
+  return { librairies: data }
+})
+```
+
+Then call your api route from any vue file:
+
+```ts [pages/index.vue]
+const callSupabaseClientInServerRoute = async () => {
+  const { librairies } = await $fetch('/api/librairies')
+}
+```
+
+### `serverSupabaseUser`
+
+This function is working similary as the [useSupabaseUser](/usage#usesupabaseuser) composable but can be used in server routes.
+
+Define your server route and just use the `serverSupabaseUser` like:
+
+```ts [server/api/me.ts]
+export default eventHandler(async (event) => {
+  const user = await serverSupabaseUser(event)
+
+  return { user }
+})
+```
+
+Then call your api route from any vue file:
+
+
+```ts [pages/index.vue]
+const getSupabaseUserFromServerRoute = async () => {
```

Comment:
Same here, only showing the `$fetch` shall be enough

Reply:
Or at least I would write:

const getSupabaseUserFromServerRoute = () => $fetch('/api/me')